src/Illuminate/Support/Fluent.php

```diff
@@ -7,19 +7,23 @@
 use Illuminate\Contracts\Support\Jsonable;
 use JsonSerializable;
 
+/**
+ * @template TKey of array-key
+ * @template TValue
+ */
 class Fluent implements Arrayable, ArrayAccess, Jsonable, JsonSerializable
 {
     /**
      * All of the attributes set on the fluent instance.
      *
-     * @var array
+     * @var array<TKey, TValue>
      */
     protected $attributes = [];
 
     /**
      * Create a new fluent instance.
      *
-     * @param  array|object  $attributes
+     * @param  array<TKey, TValue>|iterable<TKey, TValue>|object  $attributes
```

Comment:
`iterable` is a supertype of `array`, so wouldn't `iterable<TKey, TValue>|object` be sufficient?

Reply:
Yes - I've fixed that.